set-integration-testing/src/integration-test/java/eu/europeana/api/set/integration/web/WebUserSetRestIT.java

```diff
@@ -98,6 +99,17 @@ public void create_published_UserSet_400_bad_request_InvalidInput() throws Excep
         .andExpect(status().isBadRequest());
   }
 
+  @Test
+  public void create_UserSet_InvalidItems() throws Exception {
+    String requestJson = getJsonStringInput(USER_SET_INVALID_ITEMS);
+    mockMvc
+        .perform(
+            post(BASE_URL).param(CommonApiConstants.QUERY_PARAM_PROFILE, LdProfiles.MINIMAL.name())
+                .content(requestJson).header(HttpHeaders.AUTHORIZATION, regularUserToken)
+                .header(HttpHeaders.CONTENT_TYPE, MediaType.APPLICATION_JSON_VALUE))
+        .andExpect(result -> assertTrue(result.getResolvedException() instanceof ItemValidationException));
```

Comment:
There is no need for this, since we check the exact exception that is thrown, which is much more detailed than 400 status check and it btw includes that as well.

Reply:
for the use in the portal, the status code is very important, please add the check for the status

Reply:
Ok, added.